tikv/server.go

```diff
@@ -16,6 +16,12 @@ import (
 	"golang.org/x/net/context"
 )
 
+var enableShardKey = false
+
+func EnableShardKeySupport() {
```

Comment:
Could you add a function IsShardingEnabled()? And call it instead of using enableShardKey directly.